app/uk/gov/hmrc/teamsandrepositories/connectors/JenkinsConnector.scala

```diff
@@ -44,7 +44,7 @@ class JenkinsConnector @Inject()(
   private val authorizationHeader =
     s"Basic ${BaseEncoding.base64().encode(s"${config.username}:${config.token}".getBytes("UTF-8"))}"
 
-  def triggerBuildJob(baseUrl: String)(implicit ec: ExecutionContext): Future[Unit] = {
+  def triggerBuildJob(baseUrl: String)(implicit ec: ExecutionContext): Future[HttpResponse] = {
```

Comment:
`HttpResponse` shouldn't escape the Connector layer - can we extract what we need from it and return that?

Reply:
Done